Libraries/TVector/TVector.h

```diff
@@ -0,0 +1,151 @@
+#ifndef TVector_h
+#define TVector_h
+#include "Tile.h"
+
+template <class T>
+class TVector {
+  private:
+    uint8_t iSize;
+    T *vector;
+
+  public:
+    TVector();//constructor with 1 element
+    TVector(const TVector &V);//constructor that copies another vector
+    ~TVector();
+
+    uint8_t getSize();
+
+    void pushTop(T t);
+    void pushFront(T t);
+    void popTop();
+    void popFront();
```

Comment:
Maybe just me, but don't you think Top and Front are confusing names when used together?